.prettierignore

```diff
@@ -1,4 +1,5 @@
 coverage
 dist
 website/.docusaurus
-website/build
\ No newline at end of file
+website/build
+pnpm-lock.yaml
```

Comment:
I'm confused by this line. [_pnpm-lock.yaml_ should always be committed](https://pnpm.io/git#lockfiles).

When I run `pnpm install` locally, my lockfile is radically different from the one in this branch.

Reply:
this is `.prettierignore`, not `.gitignore` 
what version are you actually using with pnpm ?

Reply:
Ah, I misread the filename.

> what version are you actually using with pnpm ?
```bash
❯ pnpm --version
9.1.1
```

Reply:
I'm not sure how helpful this is, but I noticed my local lockfile version is 9.0, whereas the one initially committed is 6.0:

![Screenshot 2024-05-17 at 6 02 38 PM](https://github.com/RetroAchievements/api-js/assets/3984985/25900848-024b-4661-8946-4ff37e0d1146)

Reply:
This should be fixed now.
When i updated husky, i did it with older version of pnpm and made a regresion.